nautobot/extras/views.py

```diff
@@ -2091,3 +2091,18 @@ class WebhookDeleteView(generic.ObjectDeleteView):
 class WebhookBulkDeleteView(generic.BulkDeleteView):
     queryset = Webhook.objects.all()
     table = tables.WebhookTable
+
+
+#
+# Job Extra Views
+#
+# NOTE: Due to inheritance, JobObjectChangeLogView and JobObjectNotesView can only be
+# constructed below # ObjectChangeLogView and ObjectNotesView.
+
+
+class JobObjectChangeLogView(ObjectChangeLogView):
+    base_template = "extras/job_detail.html"
+
+
+class JobObjectNotesView(ObjectNotesView):
+    base_template = "extras/job_detail.html"
```

Comment:
As an alternative, could we maybe rename `job_detail.html` to `job_retrieve.html`? I think then these views would work without subclassing being needed?

Reply:
Nope it did not work,

Reply:
```python
def get_base_template(base_template, model):
    """
    Returns the name of the base template, if the base_template is not None
    Otherwise, default to using "<app>/<model>.html" as the base template, if it exists.
    Otherwise, check if "<app>/<model>_retrieve.html" used in `NautobotUIViewSet` exists.
    If both templates do not exist, fall back to "base.html".
    """
    if base_template is None:
        base_template = f"{model._meta.app_label}/{model._meta.model_name}.html"
        # TODO: This can be removed once an object view has been established for every model.
        try:
            get_template(base_template)
        except TemplateDoesNotExist:
            base_template = f"{model._meta.app_label}/{model._meta.model_name}_retrieve.html"
            try:
                get_template(base_template)
            except TemplateDoesNotExist:
                base_template = "base.html"
    return base_template
```

Reply:
It first determines whether `extras/jobs.html` exists, which it does. 
I'm not sure if addressing this would result in a breaking change.

Reply:
Got it, thanks for the analysis.